### DISCUSSION TITLE
error-stack: Why does `Report` implement `Serialize` but not `Deserialize`?

### DISCUSSION THREAD

**Discussion by khassar00** [hashintel/hash#2712] [error-stack (Help)] [answered]
The following text is generated by automatic translation :
error_stack::Report implements serialize but does not Deserialize.
Is there a reason?

**Comment by TimDiekmann** [comment, marked as answer]
Hi @khassar00 and thanks for asking.
How would `Deserialize` be implemented? How you would get the type information? Even if you would pass a single type for the current context you would lose all other type information which are stored in the error stack.

Please share your thoughts if you know how this could be solved 🙂

**Reply by khassar00** [reply]
I see, thanks for your reply.
